frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1529,19 +1572,30 @@ impl<T: Config> Pallet<T> {
 		//   system
 		// - signed phase was complete or not started, in which case finalization is idempotent and
 		//   inexpensive (1 read of an empty vector).
+		//
+		// If there is no queued solution, there are two possibilities:
+		//
+		// - EPM has been in `Phase::Signed` for long enough blocks, try the fallback election.
```

Comment:
These docs (and perhaps the entire PR) is a bit too "Signed-phase focused". The undsigned phase is also part of the story here. 

Here's some ideas to improve this:

1. Let's coin the term "Electing" blocks as those in which `Phase::Singed` or `Phase::Unsigned`.
2. The condition is to make sure per round, we are at least `MinBlocksBeforeEmergency` in the "Electing" blocks.
3. Based on this, I would rename `MinBlocksBeforeEmergency` to `MinElectingBlocks`